bootstrap/src/u128.ml

```diff
@@ -0,0 +1,1281 @@
+include Rudiments_int
```

Comment:
`u128` was a significant development effort, to the point that it was tempting to just make the `Hash` API use `(u64 * u64)` instead of `u128`.  However, we're probably also going to need `u256` to support cryptographic hashing (SHA-256 or SHA-3-256) for uniquely identifying source code, API signatures, etc., so it felt worthwhile to do a full implementation.  `u256` will probably require extracting various `u128` code for sharing, but I'd rather not try to anticipate the particulars right now.